server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -231,6 +231,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion INTRODUCE_FAILURES_LIFECYCLE = def(9_065_0_00);
     public static final TransportVersion PROJECT_METADATA_SETTINGS = def(9_066_00_0);
     public static final TransportVersion AGGREGATE_METRIC_DOUBLE_BLOCK = def(9_067_00_0);
+    public static final TransportVersion PINNED_RETRIEVER = def(9_068_0_00);
```

Comment:
just a side note - but when we backport this to 8.19, we need to pay extra attention on how we backport this transport version

Reply:
Yeah @mridula-s109 - we will create a `PINNED_RETRIEVER_8_19` transport version